### PR TITLE
Feature current telepath users

### PR DESCRIPTION
**Pull request by vEpiphyte**
- Expose a Daemon api to get serialized information about current sessions.
- Expose a Cell api to get the newly exposed daemon API data about current sessions (and their users).
- Include connection information (source IP, connection type, etc)
- Add an ipv6 test for telepath (executes in osx tests)

This does **not** expose the telepath session identifier so a telepath session cannot be hijacked.

### CODE REVIEW

**Comment by vEpiphyte** [comment]
This doesn't consider *httpapi* users

File: synapse/tests/test_telepath.py [hunk 1]
```diff
@@ -569,27 +596,70 @@ def __init__(self):
...
                 # test an absolute cell:// url
                 async with await s_telepath.openurl(f'cell://{path}') as prox:
                     self.eq('cell', await prox.getCellType())
+                    # unix path information is available from the session information.
+                    snfo = await cell.dmon.getSessInfo()
+                    self.eq(snfo[0].get('anfo'),
+                            {'family': 'unix',
+                             'addr': sockpath})
+
+    async def test_ipv6(self):
```
Comment (vEpiphyte) [inline synapse/tests/test_telepath.py]: This test does execute in CI in osx

File: synapse/daemon.py [hunk 2]
```diff
@@ -26,6 +26,7 @@ class Sess(s_base.Base):
         self.items = {}
         self.iden = s_common.guid()
         self.user = None
+        self.anfo = {}
```
Comment (jnwatson) [inline synapse/daemon.py]: conninfo

File: synapse/daemon.py [hunk 3]
```diff
@@ -36,6 +37,16 @@ def setSessItem(self, name, item):
     def popSessItem(self, name):
         return self.items.pop(name, None)
 
+    def pack(self):
+        ret = {'items': {name: f'{item.__module__}.{item.__class__.__name__}' for name, item in self.items.items()},
+               'anfo': self.anfo,
```
Comment (jnwatson) [inline synapse/daemon.py]: 'anfo' -> conninfo

File: synapse/tests/test_telepath.py [hunk 4]
```diff
@@ -172,12 +172,25 @@ class TeleTest(s_t_utils.SynTest):
...
-            addr = await dmon.listen('tcp://127.0.0.1:0')
             dmon.share('foo', foo)
 
             await self.asyncraises(s_exc.BadUrl, s_telepath.openurl('noscheme/foo'))
 
-            prox = await s_telepath.openurl('tcp://127.0.0.1/foo', port=addr[1])
+            prox = await s_telepath.openurl('tcp://127.0.0.1/foo', port=dmon.addr[1])
+
+            # Some bookkeeping data about the connection is available
+            # from the daemon related to the sesssion's objects and
```
Comment (jnwatson) [inline synapse/tests/test_telepath.py]: sesssion

File: synapse/tests/test_telepath.py [hunk 5]
```diff
@@ -356,15 +367,20 @@ def test_telepath_sync_genr_break(self):
             async with await s_telepath.openurl(f'ssl://{hostname}/foo', port=addr[1]) as prox:
                 self.eq(30, await prox.bar(10, 20))
 
+                # The daemon's session information for a TLS link
+                # it's own family.
```
Comment (jnwatson) [inline synapse/tests/test_telepath.py]: its

**Pull request merged**